passport-anonymous/index.d.ts

```diff
@@ -0,0 +1,10 @@
+// Type definitions for passport-anonymous 1.0
+// Project: https://github.com/jaredhanson/passport-anonymous
+// Definitions by: Pavel Puchkov <https://github.com/0x6368656174>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import passport = require("passport");
+
+declare class Strategy implements passport.Strategy {
```

Comment:
I did it like other passport packages:
- passport-beam: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/passport-beam/index.d.ts#L12-L16
- passport-facebook: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/passport-facebook/index.d.ts#L32-L37
- passport-http: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/passport-http/index.d.ts#L45-L51
- passport-local: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/passport-local/index.d.ts#L37-L44
- passport-google-oauth: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/passport-google-oauth/index.d.ts#L65-L70
- others....